Tools/GNUMake/comps/intel.mak

```diff
@@ -1,78 +1,10 @@
-#
-# Generic setup for using Intel
-#
-CXX = icpc
-CC  = icc
-FC  = ifort
-F90 = ifort
-
-CXXFLAGS =
-CFLAGS   =
-FFLAGS   =
-F90FLAGS =
-
-########################################################################
-
-intel_version = $(shell $(CXX) -dumpversion)
-
-COMP_VERSION = $(intel_version)
-
-########################################################################
-
-ifeq ($(DEBUG),TRUE)
-
-  CXXFLAGS += -g -O0 -traceback -Wcheck
-  CFLAGS   += -g -O0 -traceback -Wcheck
-  FFLAGS   += -g -O0 -traceback -check bounds,uninit,pointers
-  F90FLAGS += -g -O0 -traceback -check bounds,uninit,pointers
+CXX = icpx
+has_intel_llvm_compiler := $(shell $(CXX) -dumpversion)
 
```

Comment:
This doesn't seem like a particularly robust way to do this check, but I assume it's working for our situations of interest at the moment. We should keep an eye out, though.